pkg/modelconv/modelconv.go

```diff
@@ -3,3 +3,10 @@ package modelconv
 import "github.com/iver-wharf/wharf-core/pkg/logger"
 
 var log = logger.NewScoped("WHARF")
+
+func fallbackString(value, fallback string) string {
+	if value == "" {
```

Comment:
Can it be nil?

Reply:
Nope, strings cannot be nil in Go.

Pointers to strings can be, but `value` here is just a regular string